src/SOS/Strike/strike.cpp

```diff
@@ -1231,6 +1237,7 @@ DECLARE_API(DumpMT)
     table.WriteRow("ComponentSize:", PrefixHex(vMethTable.ComponentSize));
     table.WriteRow("DynamicStatics:", vMethTable.bIsDynamic ? "true" : "false");
     table.WriteRow("ContainsPointers:", vMethTable.bContainsPointers ? "true" : "false");
+    // FIXME: "Slots in VTable" is a misnomer. This is the number of slots in the MethodTable
```

Comment:
> slots in the MethodTable

The methods slots included in this number do not have to be physically in the MethodTable. Some of them can be non-VTable slots that are stored physically in the MethodDesc.

`Number of Methods` is probably the best description of this number. It is `number_of_inherited_methods+number_of_introduced_virtual_methods+number_of_non_virtual_methods`. The slot is basically just an index of the method in this logical sequence. Also, the number does not include EnC added methods. 

A separate question is whether this is a useful number to have. The runtime only uses for some asserts and for pre-allocating a cache that needs to fit all above methods. I see SOS uses it to enumerate methods, so I guess we can keep it in the contract for that, at least for now. But there are probably better ways to design enumerator of methods on given type.

Reply:
> I see SOS uses it to enumerate methods, so I guess we can keep it in the contract for that, at least for now. 

Actually, David's change https://github.com/dotnet/runtime/pull/101580/files#r1653880772 is going to break this method enumeration scheme.